ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-main.yaml

```
@@ -1,7 +1,22 @@
+base_images:
+  metrics-linter:
+    name: prom-metrics-linter
+    namespace: ci
+    tag: 0.0.2
 build_root:
   project_image:
     dockerfile_path: ci-builder/Dockerfile
 images:
+- dockerfile_literal: |
+    FROM src
+    COPY prom-metrics-linter /usr/bin/prom-metrics-linter
+    RUN dnf install -y jq && dnf clean all
+  from: src
+  inputs:
+    metrics-linter:
+      paths:
+        - destination_dir: .
+          source_path: .
```

I think it should be:
```suggestion
        - destination_dir: .
          source_path: /prom-metrics-linter
```

---

I'm not sure actually, relied on the dot in https://github.com/kubevirt/monitoring/blob/main/test/metrics/prom-metrics-linter/Dockerfile#L12.

---

missing `to:` field with the required image name.

---

Added `to:`

---

Done